frontend/src/common/components/FormDialog.js

```diff
@@ -1,76 +1,87 @@
-	children,
-	close,
-	maxWidth,
-	open,
-	onCancel,
-	onSubmit,
-	primaryButtonLabel,
-	secondaryButtonLabel,
-	title,
-	validationSchema
+  children,
+  close,
+  maxWidth,
+  open,
+  onCancel,
+  onSubmit,
+  primaryButtonLabel,
+  secondaryButtonLabel,
+  title,
+  validationSchema,
 }) {
+  const classes = useStyles();
 
-	const generateErrors = (values) => ValidationService(values, validationSchema).errors;
+  const generateErrors = (values) =>
+    ValidationService(values, validationSchema).errors;
 
-	return (
-		<Dialog fullWidth maxWidth={maxWidth} open={open} onClose={close}>
-			<DialogTitle>{title}</DialogTitle>
-			<DialogContent>
-				<Form
-					onSubmit={onSubmit}
-					validate={generateErrors}
-					render={({ handleSubmit }) => (
-						<form onSubmit={handleSubmit} noValidate>
-							{children}
-							<Grid
-								container
-								alignItems="center"
-								justify="flex-end"
-								spacing={1}
-							>
-								<Grid item xs={12} sm={4}>
-									<Button
-										onClick={onCancel}
-										color="secondary"
-										variant="contained"
-										fullWidth
-									>
-										{secondaryButtonLabel}
-									</Button>
-								</Grid>
-								<Grid item xs={12} sm={4}>
-									<Button
-										type="submit"
-										color="primary"
-										variant="contained"
-										fullWidth
-									>
-										{primaryButtonLabel}
-									</Button>
-								</Grid>
-							</Grid>
-						</form>
-					)}
-				/>
-			</DialogContent>
-		</Dialog>
-	);
+  return (
+    <Dialog fullWidth maxWidth={maxWidth} open={open} onClose={close}>
+      <DialogTitle>{title}</DialogTitle>
+      <DialogContent>
+        <Form
+          onSubmit={onSubmit}
+          validate={generateErrors}
+          render={({ handleSubmit }) => (
+            <form onSubmit={handleSubmit} noValidate>
+              {children}
+              <Grid
+                container
+                alignItems="center"
+                justify="flex-end"
+                spacing={1}
+              >
+                <Grid item xs={12} sm={4}>
+                  <Button
+                    onClick={onCancel}
+                    color="secondary"
+                    variant="contained"
+                    fullWidth
+                    className={classes.dialogActions}
+                  >
+                    {secondaryButtonLabel}
+                  </Button>
+                </Grid>
+                <Grid item xs={12} sm={4}>
+                  <Button
+                    type="submit"
+                    color="primary"
+                    variant="contained"
+                    fullWidth
+                    className={classes.dialogActions}
+                  >
+                    {primaryButtonLabel}
+                  </Button>
+                </Grid>
+              </Grid>
+            </form>
+          )}
+        />
+      </DialogContent>
+    </Dialog>
+  );
 }
 
 export default FormDialog;
```

Comment:
Nothing new in this form other than some spacing. Just fixing the tab spacing from my shitty previous linting.